### PR TITLE
fix: updated fonts.

### PR DESCRIPTION
**Pull request by Tulun**
Font definitions adjusted to match Umaverse.

Signed-off-by: Tulun <jaykiraly@gmail.com>

### CODE REVIEW

**Comment by daywiss** [comment]
these shoudl show before/after pics if possible

**Review by daywiss — COMMENTED** [review]
code looks good, but its really a pain to try and QA this because yyou need a contract in the right state.

File: src/components/global-styles/GlobalStyles.tsx [hunk 1]
```diff
@@ -5,33 +5,30 @@ export const typography = css`
...
   @font-face {
     font-family: "Halyard";
+    src: url("/fonts/HalyardDisplay-Regular.woff2") format("woff2"),
+      url("/fonts/HalyardDisplay-Regular.woff") format("woff");
+    font-weight: normal;
     font-style: normal;
-    font-weight: 400;
-    font-display: fallback;
-    src: url("/fonts/HalyardDisplay-Regular.woff2") format("woff2");
-    unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02BB-02BC, U+02C6, U+02DA,
```
Comment (Gamaranto) [inline src/components/global-styles/GlobalStyles.tsx:12]: I think this should be kept in, what it does is it fetches only latin characters for this font, reducing bundle size

**Review by Gamaranto — APPROVED** [review]

**Review by Gamaranto — COMMENTED** [review]
a small comment, otherwise looks good to me

File: src/components/global-styles/GlobalStyles.tsx (hunk 1 above)
Comment (Tulun) [reply src/components/global-styles/GlobalStyles.tsx:12]: Sounds good. I just pilfered what we had in Umaverse, but I can keep the unicode range.

**Pull request merged**